### PR TITLE
feat: e2e demo script

### PR DESCRIPTION
**Pull request by redlarva**
### Commit Message

```bash
feat(test) : e2e demo setup
```

see the [guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#commit-messages) for commit messages.

### Changelog Entry

```bash
version: <log entry>
```

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have documented my code in accordance with the [documentation guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#documentation)
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have run the unit tests
- [ ] I only have one commit (if not, squash them into one commit).
- [ ] I have a descriptive commit message that adheres to the [commit message guidelines](https://github.com/icon-project/community/blob/main/guidelines/technical/software-development-guidelines.md#commit-messages)

> Please review the [CONTRIBUTING.md](/CONTRIBUTING.md) file for detailed contributing guidelines.

### CODE REVIEW

File: test/chains/icon/localnet.go [hunk 1]
```diff
@@ -357,11 +380,11 @@ func (c *IconLocalnet) SetupIBC(ctx context.Context, keyName string) (context.Co
 	c.IBCAddresses = contracts.ContractAddress
 
 	params := `{"name": "test","country": "KOR","city": "Seoul","email": "prep@icon.foundation.com","website": "https://icon.kokoa.com","details": "https://icon.kokoa.com/json/details.json","p2pEndpoint": "localhost:9080"}`
-	_, _ = c.executeContract(ctx, "cx0000000000000000000000000000000000000000", "gochain", "registerPRep", params)
+	_, _ = c.executeContract(ctx, "cx0000000000000000000000000000000000000000", interchaintest.IBCOwnerAccoun, "registerPRep", params)
```
Comment (AntonAndell) [inline test/chains/icon/localnet.go:383]: IBCOwnerAccoun -> IBCOwnerAccount

**Review by AntonAndell — CHANGES_REQUESTED** [review]
Just a veri minor comment

**Review by ibrizsabin — APPROVED** [review]

File: test/chains/icon/localnet.go (hunk 1 above)
Comment (redlarva) [reply test/chains/icon/localnet.go]: fixed

**Review by AntonAndell — APPROVED** [review]
Looks good

**Review by ibrizsabin — APPROVED** [review]

**Pull request merged**